controllers/dscinitialization/servicemesh_setup.go

```diff
@@ -130,3 +142,47 @@ func configureServiceMeshFeatures() feature.FeaturesProvider {
 		return nil
 	}
 }
+
+func IsDefaultAudiences(specAudiences *[]string) bool {
```

Comment:
Maybe worth moving this piece to its own file

Reply:
think this is a good idea, moved them all to utils

e38a8ad